src/commands/data/export/resume.ts

```diff
@@ -79,6 +79,11 @@ export default class DataExportResume extends SfCommand<DataExportResumeResult>
       ms.goto('exporting records', { state: jobInfo.state });
     });
 
+    queryJob.on('error', (err) => {
+      ms.stop('failed');
+      throw err;
+    });
```

Comment:
noticed this was missing, it should make sf throw if polling/fetching records in `exportRecords` fails.